ibm_db2.c

```diff
@@ -2354,12 +2368,13 @@ static int _php_db2_connect_helper( INTERNAL_FUNCTION_PARAMETERS, conn_handle **
         /* Check if we already have a connection for this userID & database combination */
         if (isPersistent) {
             zend_resource *entry;
-            /* 9 = strlen("__db2_") + strlen('.') + strlen(".") + null */
-            hKeyLen = strlen(database) + strlen(uid) + strlen(password) + 9;
+            /* 17 = strlen("FFFFFFFF") strlen("__db2_") + strlen('.') + strlen(".") + null */
```

Comment:
```suggestion
            /* 17 = strlen("FFFFFFFF") + strlen("__db2_") + strlen('.') + strlen(".") + null */
```

Reply:
FYI `sizeof` a string literal will include the length of the string as well as the null terminator. No need to make it a comment. (Just can't assign it to a variable first, eg. this doesn't work: `const char* foo = "bar"; size_t len = sizeof(foo);`